openlibrary/catalog/add_book/__init__.py

```diff
@@ -146,6 +146,15 @@ def new_work(q, rec, cover_id):
     return w
 
 def load_data(rec):
+    """
+    Creates a new Edition,
+    searches for an existing Work,
+      creates a new one if required,
+      otherwise adds to existing Work,
+        possibly with modification.
+    :param dict rec: Edition record to load (now we have established it should be added)
+    :rtype: dict {"success": bool, "error": string | "work": {"key": <key>, "status": "created" | "modified" | "matched"} , "edition": {"key": <key>, "status": "created"}}
```

Comment:
There's a typo after "error" field; should that `|` be a `,`?

Reply:
Hrm, I meant to put that there, but it's not clear. I meant to show that you either get an `error` OR `work` + `edition`